test/rules/naming/modifier-name-mixedcase.js

```diff
@@ -23,4 +23,23 @@ describe('Linter - modifier-name-mixedcase', () => {
 
     assert.equal(report.errorCount, 0)
   })
+
+  describe('with $ character', () => {
+    const WITH_$ = {
+      $: contractWith('modifier $(address a) { }'),
+      'starting with $': contractWith('modifier $ownedBy(address a) { }'),
+      'containing a $': contractWith('modifier owned$By(address a) { }'),
+      'ending with $': contractWith('modifier ownedBy$(address a) { }'),
+    }
+
+    for (const [key, code] of Object.entries(WITH_$)) {
+      it(`should not raise func name error for functions ${key}`, () => {
```

Comment:
==> for modifiers
(wrong description on IT statement)

Reply:
```suggestion
      it(`should not raise func name error for modifiers ${key}`, () => {
```